### PR TITLE
docs: remove outdated tv docs

### PR DESCRIPTION
**Pull request by moskalakamil**
Since React Native for TV has been moved to a separate repository, information on how to develop for this library should not be included in the main React Native documentation. We should either remove this section entirely or add a link redirecting users to the react-native-tvos Wiki.

### CODE REVIEW

**Review by cortinico — APPROVED** [review]

**Comment by cortinico** [comment]
@Simek any thoughts on this one?

**Comment by Simek** [comment]
Let's make sure that @douglowder is fine with that change, and if wiki is a right place to redirect people.

**Comment by moskalakamil** [comment]
@Simek the official react-native-tvos documentation links to their wiki, thats why I decided to add this link

**Comment by Simek** [comment]
I have in mind that we might want to split up the redirected group, between those, who use frameworks and those who are not, like we do in new setup environment sections. 

For example, for Expo users, we got a dedicated guide on how to build TV apps - https://docs.expo.dev/guides/building-for-tv/.

Also, RN TV wiki seems to contains only one doc page, and rest are an external links do different resources, so it might be more valuable to link people into the doc file directly, which will guide them through the setup.

**Comment by moskalakamil** [comment]
@Simek Sure, provide me please information on which of these two options we prefer. I would also like to note that if we redirect to TV Wiki, we won't have to worry about updating this subpage in the future, as all TV-related information will be then updated by the react-native-tvos community in their repo

File: docs/building-for-tv.md [hunk 1]
```diff
@@ -4,190 +4,8 @@ title: Building For TV Devices
...
-  - `TextInput` components do not work for now (i.e. they cannot receive focus automatically, see [this comment](https://github.com/facebook/react-native/pull/16500#issuecomment-629285638)).
-    - It is however possible to use a ref to manually trigger `inputRef.current.focus()`.
-    - You can wrap your input inside a `TouchableWithoutFeedback` component and trigger focus in the `onFocus` event of that touchable. This enables opening the keyboard via the arrow keys.
-    - The keyboard might reset its state after each keypress (this might only happen inside the Android TV emulator).
-  - The content of `Modal` components cannot receive focus, see [this issue](https://github.com/facebook/react-native/issues/24448) for details.
-
-</TabItem>
-<TabItem value="tvos">
-
 > **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos) repository.
```
Comment (douglowder) [inline docs/building-for-tv.md:9]:
```suggestion
> **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos) repository. Please see the README there for information on projects for Apple TV or Android TV.
```

File: docs/building-for-tv.md [hunk 2]
```diff
@@ -4,190 +4,8 @@ title: Building For TV Devices
...
-
-- _TabBarIOS behavior_: The `TabBarIOS` component wraps the native `UITabBar` API, which works differently on Apple TV. To avoid jittery re-rendering of the tab bar in tvOS (see [this issue](https://github.com/facebook/react-native/issues/15081)), the selected tab bar item can only be set from JavaScript on initial render, and is controlled after that by the user through native code.
-
-- _Known issues_:
-
-  - [ListView scrolling](https://github.com/facebook/react-native/issues/12793). The issue can be worked around by setting `removeClippedSubviews` to false in ListView and similar components. For more discussion of this issue, see [this PR](https://github.com/facebook/react-native/pull/12944).
-
-</TabItem>
-</Tabs>
+**To get full React Native for Tv documentation, visit [react-native-tvOS Wiki](https://github.com/react-native-tvos/react-native-tvos/wiki)**
```
Comment (douglowder) [inline docs/building-for-tv.md:11]:
```suggestion
```

Comment (douglowder) [reply docs/building-for-tv.md:11]: This Wiki is not fully up to date and does not provide full documentation, so we should omit this change.

File: docs/building-for-tv.md (hunk 1 above)
Comment (douglowder) [reply docs/building-for-tv.md:9]: I added this note directing developers to the README in the TV repo, which is kept up to date (including a pointer to Expo TV docs).

**Review by douglowder — COMMENTED** [review]
See my comments 😄 and thanks for doing this!

**Comment by moskalakamil** [comment]
@douglowder done ;))

**Comment by douglowder** [comment]
>  done ;)) Should i update earlier docs versions?

@moskalakamil yes thanks, just the versions that Meta currently supports (I think that is 0.71 and up)

**Comment by moskalakamil** [comment]
@douglowder finished, thanks for guidance!

**Comment by douglowder** [comment]
@moskalakamil no problem! I approve the request, although I don't have access to do that in GitHub 😄

File: docs/building-for-tv.md [hunk 3]
```diff
@@ -4,190 +4,6 @@ title: Building For TV Devices
...
-
-- _TabBarIOS behavior_: The `TabBarIOS` component wraps the native `UITabBar` API, which works differently on Apple TV. To avoid jittery re-rendering of the tab bar in tvOS (see [this issue](https://github.com/facebook/react-native/issues/15081)), the selected tab bar item can only be set from JavaScript on initial render, and is controlled after that by the user through native code.
-
-- _Known issues_:
-
-  - [ListView scrolling](https://github.com/facebook/react-native/issues/12793). The issue can be worked around by setting `removeClippedSubviews` to false in ListView and similar components. For more discussion of this issue, see [this PR](https://github.com/facebook/react-native/pull/12944).
-
-</TabItem>
-</Tabs>
+> **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos) repository. Please see the _README_ there for information on projects for Apple TV or Android TV.
```
Comment (Simek) [inline docs/building-for-tv.md:9]:
```suggestion
> **Deprecated.** TV support has moved to the [React Native for TV](https://github.com/react-native-tvos/react-native-tvos?tab=readme-ov-file#react-native-tvos) repository. Please see the _README_ there for information on projects for Apple TV or Android TV.
```

Can we link directly to the README file?

**Review by Simek — APPROVED** [review]
Thanks for the Feedback Doug, and @moskalakamil for the updates! 🙏 

Left one last suggestion about the link, but but we also can ship as-is.

**Comment by moskalakamil** [comment]
@Simek updated 👍

**Comment by moskalakamil** [comment]
@Simek updated 👍

**Comment by moskalakamil** [comment]
Hey @Simek, just following up on my PR — it’s been sitting for a while. Thanks!

**Comment by Simek** [comment]
Oh, sorry for missing that one, and thanks for the ping! 🙏 

One last thing before shipping, can we also update the file for 0.75 release, so we got a full coverage?
* https://github.com/facebook/react-native-website/blob/main/website/versioned_docs/version-0.75/building-for-tv.md

**Comment by moskalakamil** [comment]
Sure! I’ve done this @Simek

**Pull request merged**